tron/bin/action_runner.py

```diff
@@ -73,6 +73,24 @@ def validate_output_dir(path):
             raise OSError("Could not create output dir %s" % path)
 
 
+def build_environment(run_id):
+    try:
+        namespace, job, run_num, action = run_id.split('.', maxsplit=3)
+    except ValueError:
+        # if we can't parse the run_id, we don't want to abort, so just
+        # set these semi-arbitrarily
+        namespace, job, run_num, action = '', '', '', run_id
```

Comment:
`UNKOWN` would be ok I think. That would make it more clear that there was a bug in parsing.

Setting the action to the `run_id` sound bug-prone.